### PR TITLE
Alignment matrix tests

### PR DESCRIPTION
**Pull request by richarda23**
- tests for substitution matrices
- javadoc and some reformatting
- also removed some tests that required java 11 and broke Java 8 tests, from a previous PR
-

### CODE REVIEW

File: biojava-core/src/test/java/org/biojava/nbio/core/util/CRC64ChecksumTest.java
Comment (josemduarte) [inline biojava-core/src/test/java/org/biojava/nbio/core/util/CRC64ChecksumTest.java:77]: Could you explain why this one is removed?

**Review by josemduarte — APPROVED** [review]
LGTM, thanks! I only have one question below.

In any case before merging we'll need to get the CI problems fixed. Travis is not running currently, see #958

File: biojava-core/src/test/java/org/biojava/nbio/core/util/CRC64ChecksumTest.java
Comment (richarda23) [reply biojava-core/src/test/java/org/biojava/nbio/core/util/CRC64ChecksumTest.java:77]: because it's a method added in Java9 so won't compile in Java 8. I recently moved from Eclipse to IntelliJ and assumed it was running tests according to Maven setup, sorry I should have just run tests from maven command line before originally committing this.

**Comment by josemduarte** [comment]
The CI problems are still not solved. I'll merge this after testing it locally.

**Pull request merged**
